lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_EnsembleReordering.py

```diff
@@ -542,9 +542,10 @@ def setUp(self):
             add_forecast_reference_time_and_forecast_period(
                 set_up_temperature_cube()))
         self.post_processed_percentiles.coord("realization").rename(
-            "percentile")
-        self.post_processed_percentiles.coord("percentile").points = (
-            [0.1, 0.5, 0.9])
+            "percentile_over_realization")
+        self.post_processed_percentiles.coord(
+            "percentile_over_realization").points = (
+                [0.1, 0.5, 0.9])
```

Comment:
This runs either way, but should these percentiles be *100 as in other routines?